### ISSUE TITLE
Accessibility: email in contact form does not have corresponding label

### ISSUE THREAD

**Issue by bregman90** [usdigitalresponse/election-websites#167]
Discovered via: https://wave.webaim.org/report#/https://elections.usdr.dev/about-us/

Possibly because the label for reads "contact-name" instead of "contact-email"?

<img width="1345" alt="Screen Shot 2020-10-07 at 10 54 00 AM" src="https://user-images.githubusercontent.com/4401580/95368834-7fa61800-088b-11eb-845d-71b9039cba59.png">
